docs/source/docs/registry.md

```diff
@@ -0,0 +1,31 @@
+# Registry
+
+> This is a functionality under active development. A full documentation will be
+> released later.
+
+DeepReg is adapting the usage of the registry system recently, to facilitate the usage
+of custom functionalities including
+
+- neural network architecture
+- loss
+- optimizer
+- data pre-processing
+- data loader
+
+The registry is defined in `deepreg/registry.py`, where the class `Registry` maintains a
+dictionary mapping `(category, key)` to `value`. It also provides the
+`build_from_config` functionality, which allows creating a class instance from a config
+directly. This allows the simplification of the config so that each configuration file
+need to provide the name and necessary arguments.
+
+With the registry, when developing new classes **inside DeepReg**, we should use the
+corresponding decorator, so that the class is registered, cf.
```

Comment:
why cf.?

Reply:
I don't understand this either

Reply:
https://en.wikipedia.org/wiki/Cf.#:~:text=The%20abbreviation%20cf.,with%20the%20topic%20being%20discussed.